PWGDQ/DataModel/ReducedInfoTables.h

```diff
@@ -401,6 +401,32 @@ using Dilepton = Dileptons::iterator;
 using DileptonExtra = DileptonsExtra::iterator;
 using DimuonAll = DimuonsAll::iterator;
 
+// candidate information
+namespace reducedcandidate
+{
+DECLARE_SOA_INDEX_COLUMN(ReducedEvent, reducedevent); //!
+DECLARE_SOA_COLUMN(McDecision, mcDecision, uint32_t); //!
+DECLARE_SOA_COLUMN(Mass, mass, float);                //!
+DECLARE_SOA_COLUMN(Pt, pt, float);                    //!
+DECLARE_SOA_COLUMN(Eta, eta, float);                  //!
+DECLARE_SOA_COLUMN(Tauz, tauz, float);                //!
+DECLARE_SOA_COLUMN(Tauxy, tauxy, float);              //!
+DECLARE_SOA_COLUMN(Lz, lz, float);                    //! Longitudinal projection of decay length
+DECLARE_SOA_COLUMN(Lxy, lxy, float);                  //! Transverse projection of decay length
+} // namespace reducedcandidate
+
+DECLARE_SOA_TABLE(DQcandidates, "AOD", "RTDQCANDIDATE", //!
```

Comment:
Name of the table should be changed accordingly

Reply:
Fixed